google-datacatalog-saphana-connector/tests/google/datacatalog_connectors/saphana/test_data/metadata.json

```diff
@@ -0,0 +1 @@
+{"schemas": [{"name": "saphana", "tables": [{"columns": [{"default_value": NaN, "name": "a", "nullable": "YES", "type": "integer"}], "name": "test_temp", "type": "BASE TABLE"}, {"columns": [{"default_value": NaN, "name": "uid", "nullable": "YES", "type": "integer"}, {"default_value": NaN, "name": "mid", "nullable": "NO", "type": "integer"}, {"default_value": NaN, "name": "message", "nullable": "YES", "type": "character varying"}, {"default_value": NaN, "name": "ptime", "nullable": "YES", "type": "date"}], "name": "messages", "type": "BASE TABLE"}, {"columns": [{"default_value": NaN, "name": "uid", "nullable": "NO", "type": "integer"}, {"default_value": NaN, "name": "name", "nullable": "YES", "type": "character varying"}], "name": "users", "type": "BASE TABLE"}]}]}
```

Comment:
`NaN` is not a valid JSON token AFAIK. Is it possible to replace it and strictly follow the JSON spec?